index.js

```diff
@@ -33,8 +38,10 @@ Plugins.prototype = Emitter({
       next = opts;
       opts = null;
     }
-    argv = this.minimist(argv, opts);
-    this.plugins.run(argv, function (err, args) {
```

Comment:
I think it'll be correct to just remove the `merge` part of this code and pass the `opts` directly to `minimist` since this will be used like... `cli.parse(process.argv.slice(2), { ... })` and be like using `minimist` directly. The original `this.options` are available to the plugins when `fn(this)` is called inside `use`. If plugins need additional options for themselves, then they would use a pattern like:

```js
module.exports = function pluginWrapper (options) {
  options = options || {};
  return function plugin (cli) {
    // cli is `this` from `minimist-plugins`
    var opts = merge({}, cli.options, options);
    return function (argv, next) {
      next(null, argv);
    };
  };
};
```